singer-commons/example/loggingaudit_client_config_thriftlogger_example.properties

```diff
@@ -0,0 +1,42 @@
+###############################################################################
+#
+# config for loggingaudit client
+#
+###############################################################################
+
+
+stage=thriftlogger
+enableAuditForAllTopicsByDefault=false
+queueSize=100000
+enqueueWaitInMilliseconds=0
+
+sender.type=kafka
+sender.kafka.topic=logging_audit
+sender.kafka.producerConfig.bootstrap.servers=testkafka-001:9092,testkafka-002:9092,testkafka-003:9092
```

Comment:
Using producerConfig is consistent with current naming used in Singer module.  If we want to shorten the name,  this could be done after the logging-audit branch is merged. Refactoring, avoiding duplicate code and simplifying naming can be done easier.